src/bp/core/repositories/sessions.ts

```diff
@@ -60,7 +60,9 @@ export class KnexSessionRepository implements SessionRepository {
         temp_data: this.database.knex.json.set(session.temp_data || {}),
         session_data: this.database.knex.json.set(session.session_data || {}),
         modified_on: this.database.knex.date.now(),
-        created_on: this.database.knex.date.now()
+        created_on: this.database.knex.date.now(),
+        context_expiry: session.context_expiry ? this.database.knex.date.format(session.context_expiry) : eval('null'),
```

Comment:
I don't think we should ever use eval, if you really need null, you can mute the linter for that line. Can't it be undefined ?

Reply:
yeah I copy pasted this line. Previously it was possible to have a null expiry, but now I'm enforcing it. So we could change this logic and also make the column not nullable.